libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/component/sound/SoundButtonComponent.kt

```diff
@@ -0,0 +1,44 @@
+package com.mapbox.navigation.dropin.component.sound
+
+import android.view.View
+import com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI
+import com.mapbox.navigation.core.MapboxNavigation
+import com.mapbox.navigation.core.lifecycle.MapboxNavigationApp
+import com.mapbox.navigation.dropin.lifecycle.UIComponent
+import com.mapbox.navigation.ui.voice.view.MapboxSoundButton
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.channels.awaitClose
+import kotlinx.coroutines.flow.callbackFlow
+import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.launch
+
+@OptIn(ExperimentalPreviewMapboxNavigationAPI::class)
+class SoundButtonComponent(
+    private val soundButton: MapboxSoundButton,
+) : UIComponent() {
+
+    override fun onAttached(mapboxNavigation: MapboxNavigation) {
+        super.onAttached(mapboxNavigation)
+        val audioGuidance = MapboxNavigationApp.getObserver(MapboxAudioComponent::class)
```

Comment:
Eventually we may do this. The reason I'm holding off is because we have not yet settled on a philosophy for `UIBinder` implementations. For example, should we have  `ActiveGuidanceActionBinder`, and `DropInNavigationViewCoordinator` may be deleted. 

Essentially, we're optimizing for portability. `SoundButtonComponent` is easy to copy and paste anywhere if there is a single dependency.